scripts/projects.js

```diff
@@ -1,10 +1,10 @@
-// (function(module) {
+(function(module) {
   var projects = [];
 
-  function Project (projectsList) {
-    for (var keys in projectsList) {
-      this[keys] = projectsList[keys];
-    }
+  function Project (opts) {
```

Comment:
I like how you refactored this constructor function. Now multiple methods can be attached to this constructor.